.circleci/config.yml

```diff
@@ -164,6 +164,7 @@ jobs:
       - run: |
           sudo apt update && sudo apt install docker-ce cargo nodejs
```

Comment:
Ouw, i think what you're looking for is to upgrade node version to 12, not to upgrade yarn. You could do this by installing nodejs using nvm instead of from apt. Maybe something like this:
https://github.com/obserfy/vor/blob/347cb088da0cd04b19d99d892274cd54c6097557/.circleci/config.yml#L54-L61